### PR TITLE
ooniprobe: add page

### PR DESCRIPTION
**Pull request by marek22k**
OONI Probe Cli User Guide: https://ooni.org/support/ooni-probe-cli

- **Version of the command being documented (if known):** 3.18.1

### CODE REVIEW

File: pages/common/ooniprobe.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
+# ooniprobe
+
+> Open Observatory of Network Interference.
+> Test the blocking of websites and apps. Measure the speed and performance of your network.
+> More information: <https://ooni.org/>.
```
Comment (kbdharun) [inline pages/common/ooniprobe.md:5]:
```suggestion
> More information: <https://ooni.org/support/ooni-probe-cli/>.
```
I would suggest linking to the man page/docs directly here.

File: pages/common/ooniprobe.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+
+- List of all tests performed:
```
Comment (kbdharun) [inline pages/common/ooniprobe.md:7]:
```suggestion
- List all tests performed:
```

File: pages/common/ooniprobe.md [hunk 3]
```diff
@@ -0,0 +1,33 @@
+# ooniprobe
+
+> Open Observatory of Network Interference.
```
Comment (kbdharun) [inline pages/common/ooniprobe.md:3]:
```suggestion
> Open Observatory of Network Interference (OONI).
```

**Review by kbdharun — COMMENTED** [review]
Thanks for your contribution. I have some suggestions for this page.

File: pages/common/ooniprobe.md (hunk 1 above)
Comment (marek22k) [reply pages/common/ooniprobe.md]: Thanks for the suggestions. I should have implemented them all now.

**Review by kbdharun — APPROVED** [review]

**Review by acuteenvy — APPROVED** [review]

**Pull request merged**